### PR TITLE
fix: Navbar desktop and mobile

### LINKED ISSUE #8177: Navbar desktop and mobile
image
let's make it more consistent and better numbers 

image

# Changes 
- desktop navbar to 80px 
- mobile navbar to 64px from 57 or something like this 

**with mobile**, there will also be a change in the navbar logo 
from 480px, lets use the smaller icon, without the beta 
Klight V4mobile

Kdark V4mobile

there will also be a change in the menu height
image
- it should match the navbar height
- same with modal header (white part) 
image

### LINKED ISSUE #8178: Filters mobile
image
the top part looks super bad, lets solve it 
- Ideally lets make the top part the same height as the new navbar height
- #8177 

- lets also replace the x-close icon - should look same as the icons that are in the modals

apply this to all filter full screen modals

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #8177
- [x] Closes #8178

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2023-11-22 at 11 38 47

CleanShot 2023-11-22 at 11 36 02

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 7f0e2b1</samp>

This pull request improves the layout and responsiveness of the navbar and the modals that depend on it. It uses consistent variables for the navbar height and adjusts the styles of the `logo`, `connect-wallet`, `NotificationBoxModal`, `ShoppingCartModal`, and `NeoModalHead` components. It also makes the `NeoModalHead` icon clickable to close the modal.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 7f0e2b1</samp>

> _`Navbar` adapts_
> _Logo changes with the mode_
> _Autumn of design_

### CODE REVIEW

**Comment by hassnian** [comment]
@exezbcz ui check pls

File: components/shared/filters/MobileFilter.vue [hunk 1]
```diff
@@ -10,12 +10,13 @@
...
-          <p class="card-header-title has-text-weight-bold">
+        <div
+          class="filters-header border-bottom is-flex is-align-items-center is-justify-content-space-between">
+          <p class="has-text-weight-bold">
             {{ $t('general.filters') }}
           </p>
-          <a class="card-header-icon">
-            <NeoIcon icon="x" @click="onClose" />
+          <a>
+            <NeoIcon icon="xmark" @click="onClose" />
```
Comment (roiLeo) [inline components/shared/filters/MobileFilter.vue:19]: **suggestion**: put click event on `button` or `a` tag not on icon

File: libs/ui/src/components/NeoModalHead/NeoModalHead.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <template>
   <header v-if="isMobile" class="modal-card-head">
     <div class="modal-card-title" @click.stop="emit('close')">
-      <NeoIcon icon="chevron-left" />
+      <NeoIcon icon="chevron-left" class="is-clickable" />
```
Comment (roiLeo) [inline libs/ui/src/components/NeoModalHead/NeoModalHead.vue:4]: **suggestion**: put click event on `button` or `a` tag not on icon

**Review by roiLeo — COMMENTED** [review]
Search button is a bit off
Capture d’écran 2023-11-22 à 8 46 59 AM

when toggle navbar size is increased

File: components/Navbar.vue [hunk 3]
```diff
@@ -252,9 +251,11 @@ const isLandingPage = computed(
...
 )
 
-const logoSrc = computed(() =>
-  isDarkMode.value ? '/Koda_Beta_dark.svg' : '/Koda_Beta.svg',
-)
+const logoSrc = computed(() => {
+  const variant = isMobile.value ? 'Koda' : 'Koda_Beta'
+  const color = isDarkMode.value ? 'dark' : ''
+  return `/${variant}${color && `_${color}`}.svg`
+})
```
Comment (preschian) [inline components/Navbar.vue:258]: how about this?

```suggestion
const logoSrc = computed(() => {
  const variant = isMobile.value ? 'Koda' : 'Koda_Beta'
  const color = isDarkMode.value ? '_dark' : ''
  return `/${variant}${color}.svg`
})
```

File: components/shared/filters/MobileFilter.vue (hunk 1 above)
Comment (hassnian) [reply components/shared/filters/MobileFilter.vue]: fixed

File: components/Navbar.vue (hunk 3 above)
Comment (hassnian) [reply components/Navbar.vue]: fixed

File: libs/ui/src/components/NeoModalHead/NeoModalHead.vue (hunk 2 above)
Comment (hassnian) [reply libs/ui/src/components/NeoModalHead/NeoModalHead.vue]: fixed

**Review by preschian — APPROVED** [review]
code lgtm

File: components/shared/filters/MobileFilter.vue [hunk 4]
```diff
@@ -214,6 +215,15 @@ watch(() => route.query, syncFromUrl, { immediate: true })
   gap: 20px;
 }
 
+.filters-header {
+  padding: 0 1rem;
+  min-height: $navbar-mobile-min-height;
+
+  @include ktheme() {
+    color: theme('text-color');
+  }
```
Comment (Jarsen136) [inline components/shared/filters/MobileFilter.vue:224]: Is it necessary?
image

File: components/shared/filters/MobileFilter.vue [hunk 5]
```diff
@@ -214,6 +215,15 @@ watch(() => route.query, syncFromUrl, { immediate: true })
   gap: 20px;
 }
 
+.filters-header {
+  padding: 0 1rem;
```
[the remark this answers is not in the document]
Comment (hassnian) [reply components/shared/filters/MobileFilter.vue]: fixed

File: components/shared/filters/MobileFilter.vue (hunk 4 above)
Comment (hassnian) [reply components/shared/filters/MobileFilter.vue]: I was checking the prev class 
CleanShot 2023-11-22 at 17 40 15@2x

but now i see it's not needed

**Review by Jarsen136 — APPROVED** [review]

**Comment by prury** [comment]
navbar desktop 80px ✔️ 
navbar mobile 64px ✔️ 
small logo on mobile ✔️ 
x close icon on filters ✔️ 
filter top part height ✔️ 

bug: 

image

**Comment by hassnian** [comment]
@prury fixed
CleanShot 2023-11-23 at 10 49 22@2x

**Pull request merged**